frontend/index.html

```diff
@@ -2,11 +2,11 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="/images/logo.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>twp</title>
   </head>
-  <body>
+  <body style="background-color: #061e18">
```

Comment:
should this value use `var()` or it's impossible here?

Reply:
yes it actually could do so without creating/ linking css file in index.html, fixed